package.json

```diff
@@ -20,8 +20,8 @@
     "@oclif/command": "^1.5.19",
     "@oclif/config": "^1.14.0",
     "@oclif/plugin-help": "^5.1.11",
-    "@twilio-labs/serverless-api": "^4.0.3",
-    "@twilio/cli-core": "^6.7.0",
+    "@twilio-labs/serverless-api": "^5.4.0",
```

Comment:
@shrutiburman did you ensure this doesn't break any of the ahoy apps that rely on this plugin?

Reply:
Wasn't aware of the ahoy apps. Have only tested the default cmds that the plugin has. Create, deploy and delete a video app. Can you help me with the ahoy apps implementation?

Reply:
This plugin was created to to enable customers to build/deploy our Video reference apps we provide for customers. Here are instructions to deploy the React app: https://github.com/twilio/twilio-video-app-react#clone-the-repository 

I would assume upgrading will be fine but just wanted to call it out.

Reply:
Based on the [readme](https://github.com/twilio/twilio-video-app-react#deploy-the-app-to-twilio) file of the twilio-video-app-react, I did some basic acceptance testing. [Gist here](https://gist.github.com/shrutiburman/1e427538e826569835b9ca84d97f7a72). It'd be great if you or someone from the video-app-react team could take a look at it.